lib/pdk/cli/util/module_generate_helper.rb

```diff
@@ -0,0 +1,23 @@
+module PDK
+  module CLI
+    module Util
+      class ModuleGenerateHelper
+        def self.generate_module(module_name, target_dir, opts)
+          unless Util::OptionValidator.valid_module_name?(module_name)
+            error_msg = _(
+              "'%{module_name}' is not a valid module name.\n" \
+              'Module names must begin with a lowercase letter and can only include lowercase letters, digits, and underscores.',
+            ) % { module_name: module_name }
+            raise PDK::CLI::FatalError, error_msg
+          end
+
+          opts[:name] = module_name
+          opts[:target_dir] = target_dir.nil? ? module_name : target_dir
+
+          PDK.logger.info(_('Creating new module: %{modname}') % { modname: module_name })
+          PDK::Generate::Module.invoke(opts)
```

Comment:
If `PDK::Generate::Module.invoke` is horrible enough to require a helper to be called, please fix the actual API, instead of layering on!